processor/cumulativetodeltaprocessor/README.md

```diff
@@ -23,6 +23,10 @@ The following settings can be optionally configured:
 - `include`: List of metrics names or patterns to convert to delta.
 - `exclude`: List of metrics names or patterns to not convert to delta.  **If a metric name matches both include and exclude, exclude takes precedence.**
 - `max_staleness`: The total time a state entry will live past the time it was last seen. Set to 0 to retain state indefinitely. Default: 0
+- `initial_value`: Handling of the first observed point for a given metric identity. Default: `auto`.
```

Comment:
Can you add to the README why someone might choose `drop` vs `keep` vs `auto`.

Reply:
I've added some guidance.